apps/dashboard/src/components/primitives/editor.tsx

```diff
@@ -183,10 +185,10 @@ export const Editor = React.forwardRef<{ focus: () => void; blur: () => void },
         // which results in value not being updated and "jumping" effect in the editor
         // to prevent this we need to flush the state updates synchronously
         flushSync(() => {
-          onChange?.(value);
+          onChangeRef.current?.(value);
```

Comment:
The performance improvement for the CodeMirror Editor. When the `onChange` prop changes, usually on every render it has a slow performance and lag, this approach fixes it.